lib/NativeUtils.js

```diff
@@ -40,8 +39,6 @@ module.exports = class NativeUtils extends Utils {
                 this.path.currentPath = this.path.android;
             } else if (this.getConfig('platform') === 'ios') {
                 this.path.currentPath = this.path.ios;
-            } else if (this.getConfig('platform') === 'electron') {
```

Comment:
same here: probably add a comment why `electron` is missing here

Reply:
// electron is part of ./BrowserUtils.js

Reply:
// electron is part of ./BrowserUtils.js